src/Makefile.stub

```diff
@@ -26,6 +23,6 @@ default:
 
 # Better than nothing :-/
 clean:
-	+$(MAKE) -f Makefile.legacy $@
+	echo ";)"
```

Comment:
The above is not anywhere near good enough. We need to `rm *.o` at the very least.